frame/claims/src/lib.rs

```diff
@@ -7,7 +7,39 @@ mod address;
 #[cfg(feature = "std")]
 pub use address::*;
 
+mod migration {
+	// --- substrate ---
+	use frame_support::migration::*;
+	// --- darwinia ---
+	use crate::*;
+
+	pub fn migrate<T: Trait>() {
+		sp_runtime::print("Migrating DarwiniaClaims...");
+
+		if let Some(total) = take_storage_value::<RingBalance<T>>(b"DarwiniaClaims", b"Total", &[])
+		{
+			let minimum_balance = T::RingCurrency::minimum_balance();
+			let _ = T::RingCurrency::make_free_balance_be(
```

Comment:
Not sure for this usage, because didn't found sample usage like module function call in Substrate codebase. 

Need confirming.